### PR TITLE
Create api endpoint to store a new document 

### PR DESCRIPTION
**Pull request by hackermariama**

### CODE REVIEW

File: server/api/documents.js [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
+      from_language_code,
+      to_language_code,
+      submission_date,
+      due_date,
+      owner_id,
+      name,
+      format,
+      content
+    };
+    console.log(document);
```
Comment (llh1) [inline server/api/documents.js]: You can remove console.log :)

File: server/auth/routes.js [hunk 2]
```diff
@@ -88,4 +88,26 @@ router.post("/register", async (req, res, next) => {
...
+  };
+  db.createDocument(document)
+    .then(() => {
+      res.send({
+        success: true,
+        message: "Document created"
+      });
+    })
+    .catch(err => {
+      console.log(err);
```
Comment (llh1) [inline server/auth/routes.js]: You can remove console.log

File: server/db/recreate-schema.sql [hunk 3]
```diff
@@ -31,7 +31,7 @@ CREATE TABLE documents (
   name               VARCHAR(60) NOT NULL,
   from_language_code   VARCHAR(6) REFERENCES languages(code) NOT NULL,
   to_language_code     VARCHAR(6) REFERENCES languages(code) NOT NULL,
-  status           document_status,
+  status           document_status NOT NULL DEFAULT 'Waiting',
```
Comment (llh1) [inline server/db/recreate-schema.sql:35]: Great 👍

File: server/services/database/documents.js [hunk 4]
```diff
@@ -2,6 +2,41 @@ const { Pool } = require("pg");
...
+        owner_id,
+        name,
+        format,
+        content
+      ],
+      (error, result) => {
+        if (error) {
+          reject(error);
+        } else {
+          console.log(result);
```
Comment (llh1) [inline server/services/database/documents.js]: Remove console.log

File: server/services/database/documents.js [hunk 5]
```diff
@@ -44,5 +79,9 @@ const getUserDocuments = userId => {
 
 module.exports = {
   getAllDocuments,
+api-endpoint
```
Comment (llh1) [inline server/services/database/documents.js]: I think this should not be here :)

File: server/services/database/documents.js [hunk 6]
```diff
@@ -44,5 +79,9 @@ const getUserDocuments = userId => {
...
+  createDocument
+
   getUserDocuments
+ master
```
Comment (llh1) [inline server/services/database/documents.js]: Same with this, I don't think this should be written.

File: server/auth/routes.js [hunk 7]
```diff
@@ -88,4 +88,26 @@ router.post("/register", async (req, res, next) => {
     });
 });
 
+//Documents created
+
+router.post("/", async (req, res, next) => {
```
Comment (llh1) [inline server/auth/routes.js]: What is this endpoint? Should it be removed as your API endpoint is implemented in api/documents.js ?

**Review by llh1 — CHANGES_REQUESTED** [review]

File: server/api/documents.js (hunk 1 above)
Comment (hackermariama) [reply server/api/documents.js]: ok its removed

File: server/services/database/documents.js (hunk 6 above)
Comment (hackermariama) [reply server/services/database/documents.js]: it shouldnt be there,i removed it

File: server/services/database/documents.js (hunk 5 above)
Comment (hackermariama) [reply server/services/database/documents.js]: yes its removed

File: server/auth/routes.js (hunk 7 above)
Comment (hackermariama) [reply server/auth/routes.js]: yes it should be removed, this was done by mistake

**Review by NadaKartouch — APPROVED** [review]

File: server/services/database/users.js [hunk 8]
```diff
@@ -72,9 +72,17 @@ const getUserById = id => {
   });
 };
 
+const createDocument = ({ status, current_date, text, current_user_id }) => {
```
Comment (llh1) [inline server/services/database/users.js]: I cannot find where this is used, if it's not used, can you please remove it?

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/LanguageOptions.js [hunk 9]
```diff
@@ -1,4 +1,25 @@
...
+  { key: "Kor", text: "Korean", value: "Korean" },
+  { key: "Lith", text: "Lithuanian", value: "Lithuanian" },
+  { key: "Per", text: "Persian", value: "Persian" },
+  { key: "Pol", text: "Polish", value: "Polish" },
+  { key: "Por", text: "Portuguese", value: "Portuguese" },
+  { key: "Rus", text: "Russian", value: "Russian" },
+  { key: "Spa", text: "Spanish", value: "Spanish" },
+  { key: "Swe", text: "Swedish", value: "Swedish" },
+  { key: "Tur", text: "Turkish", value: "Turkish" },
+  { key: "Viet", text: "Vietnamese", value: "Vietnamese" },
```
Comment (llh1) [inline client/src/components/LanguageOptions.js]: All these languages should be removed from the PR.

**Review by llh1 — CHANGES_REQUESTED** [review]

**Review by llh1 — APPROVED** [review]

**Pull request merged**